core/src/main/java/org/pentaho/di/connections/vfs/BaseVFSConnectionProvider.java

```diff
@@ -90,4 +101,65 @@ protected static boolean getBooleanValueOfVariable( VariableSpace space, String
   protected VariableSpace getSpace( ConnectionDetails connectionDetails ) {
     return connectionDetails.getSpace() == null ? Variables.getADefaultVariableSpace() : connectionDetails.getSpace();
   }
+
+  @Override
```

Comment:
For `BaseVFSConnectionProvider#test`, what is this function overriding?

I don't see `test( T connectionDetails, VFSConnectionTestOptions connectionTestOptions ) `in either VFSConnectionProvider nor ConnectionProvider